extension-manifest-v3/src/utils/shims.js

```diff
@@ -22,6 +22,8 @@ function shimAlarms() {
     },
     async clear(name) {
       clearTimeout(alarms.get(name));
+      clearInterval(alarms.get(name));
```

Comment:
clearTimeout and clearInterval for the same value. I assume, there should be only one statement.

Reply:
It's intentional, it might be a timeout or interval. See it here and there:

timeout - https://github.com/ghostery/ghostery-extension/pull/1161/files#diff-4484ec0f55d82abb59d1c3a6bb7539a7290569072aa1a99a8c1bd66a0be70ceaR44

interval - https://github.com/ghostery/ghostery-extension/pull/1161/files#diff-4484ec0f55d82abb59d1c3a6bb7539a7290569072aa1a99a8c1bd66a0be70ceaR50

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout

> It's worth noting that the pool of IDs used by [setTimeout()](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout) and [setInterval()](https://developer.mozilla.org/en-US/docs/Web/API/setInterval) are shared, which means you can technically use clearTimeout() and [clearInterval()](https://developer.mozilla.org/en-US/docs/Web/API/clearInterval) interchangeably. However, for clarity, you should avoid doing so.

As long as browsers implement it that way, it will work (the same as if you call clearTimeout twice). But it relies on that property that the pool is shared. (If an implementation would different pools of counters if would not be correct anymore.)

If possible I would avoid it, or if absolutely needed, rather only leave one call and add a comment that it assume that the JavaScript engine uses a shared pool (which all current ones, I do I think).

Reply:
The current implementation uses both for periodical alarms. Of course, it is possible to set `timeoutId` and `intervalId` separately, but it will overcomplicate the code. I choose a comment in that situation. 

BTW there was missing clear out of the interval in `clearAll()`, so I made a fix.